Domain/Enums/FlightClassType.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+namespace Airport_Ticket_Booking.Domain.General
+{
+    public enum FlightClassType
+    {
+        Economy,
+        Business, 
+        First_Class
+    }
```

Comment:
1. using Unknown as the first value to recognize bugs. 
2. specify the value explicitly so we can be more aware of the values when adding new values.
3. use pascal case since it's mostly common in C#. You can read more about the naming conventions in C#.
```suggestion
    public enum FlightClassType
    {
        Unknown = 0,
        Economy = 1,
        Business = 2, 
        FirstClass = 3
    }
```